lms/djangoapps/certificates/api.py

```diff
@@ -575,38 +576,45 @@ def get_allowlisted_users(course_key):
     return User.objects.filter(certificatewhitelist__course_id=course_key, certificatewhitelist__whitelist=True)
 
 
-def create_certificate_allowlist_entry(user, course_key, notes):
```

Comment:
Apologies that the next section looks a bit chaotic. I tried to re-errange some of the functions so that all the allowlist functionality was located near each other.